tests/browser_context_test.go

```diff
@@ -340,6 +340,29 @@ func TestBrowserContextCookies(t *testing.T) {
 				},
 			},
 		},
+		"same_site_strict_cookie": {
+			setupHandler: func(w http.ResponseWriter, _ *http.Request) {
+				w.Header().Set("Set-Cookie", "name=value;SameSite=Strict")
+			},
+			documentCookiesSnippet: `
+				() => {
+					return document.cookie;
```

Comment:
Is there a way to set a cookie that isn't from the "same site", and then ensure that we don't see if in the response?

Reply:
This test is for testing whether we receive the `SameSite: api.CookieSameSiteStrict` field correctly. It would be nice to have such a test, but Chromium/Chrome won't let webpages set cookies in a different domain when webpages set this header. I'm afraid it would be like testing the browser's behavior as in [here](https://github.com/grafana/xk6-browser/pull/1005#discussion_r1304412292).